atom/common/api/features.cc

```diff
@@ -2,8 +2,10 @@
 // Use of this source code is governed by the MIT license that can be
 // found in the LICENSE file.
 
-#include "atom/common/node_includes.h"
 #include "native_mate/dictionary.h"
+// clang-format off
+#include "atom/common/node_includes.h"  // NOLINT(build/include_alpha)
+// clang-format on
```

Comment:
why is this ordering necessary? please add a comment :)

Reply:
Because "atom/common/node_includes.h" is special )
I agree that a comment would be nice to have here, but then all includes of this header should be documented and it can be done separately.

Reply:
The reason I called it out here is because the ordering changed, so presumably it worked fine in the other order pre-M67. What changed to break it?

Reply:
Must be some duplicate declaration or some other collision between Chromium and Node sources. I don't know.
The thing is that "node_includes.h" should always be the last include.

Reply:
Yup, @alexeykuzmin is right, there were redefinition errors with trace macros, it is always recommended to include this header at the end to avoid these collisions https://github.com/electron/electron/blob/master/atom/common/node_includes.h#L16-L33 , the fact that it worked early was just some luck.